### PR TITLE
MF-898 - Change thing's service to use bulk connect

### PR DESCRIPTION
**Pull request by nwneisen**
Signed-off-by: Nick Neisen <nwneisen@gmail.com>

### What does this do?
Add bulk connect to the thing's service.

### Which issue(s) does this PR fix/relate to?
Related to #898.

### List any changes that modify/break current functionality
Things can now be connected to a channel at a service level.

### Have you included tests for your changes?
Tests for single creation modified as needed.

### Did you document any new/modified functionality?
Documentation will be modified in the final PR when the user's endpoint is changed.

### CODE REVIEW

File: bootstrap/mocks/things.go [hunk 1]
```diff
@@ -79,11 +79,13 @@ func (svc *mainfluxThings) Connect(_ context.Context, owner, chanID, thingID str
 		return things.ErrUnauthorizedAccess
 	}
 
-	if svc.channels[chanID].Owner != userID.Value {
-		return things.ErrNotFound
-	}
+	for _, thID := range thIDs {
+		if svc.channels[chID].Owner != userID.Value {
```
Comment (manuio) [inline bootstrap/mocks/things.go]: Move this check before the loop please

**Review by manuio — CHANGES_REQUESTED** [review]

File: bootstrap/mocks/things.go (hunk 1 above)
Comment (nwneisen) [reply bootstrap/mocks/things.go]: Oops.  Thank you.

File: bootstrap/mocks/things.go [hunk 2]
```diff
@@ -70,20 +70,21 @@ func (svc *mainfluxThings) ViewThing(_ context.Context, owner, id string) (thing
...
 	defer svc.mu.Unlock()
 
 	userID, err := svc.users.Identify(context.Background(), &mainflux.Token{Value: owner})
 	if err != nil {
 		return things.ErrUnauthorizedAccess
 	}
-
-	if svc.channels[chanID].Owner != userID.Value {
+	if svc.channels[chID].Owner != userID.Value {
 		return things.ErrNotFound
```
Comment (manuio) [inline bootstrap/mocks/things.go]: I would use `ErrUnauthorizedAccess` instead of `errNotFound`

**Review by manuio — CHANGES_REQUESTED** [review]

**Review by manuio — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**